core/src/main/scala/akka/kafka/internal/ConfigSettings.scala

```diff
@@ -4,32 +4,31 @@
  */
 package akka.kafka.internal
 
-import com.typesafe.config.{Config, ConfigObject, ConfigValue}
+import java.util
+
+import com.typesafe.config.{Config, ConfigObject}
+
+import scala.collection.JavaConverters.asScalaSetConverter
 
 /**
  * INTERNAL API
  */
 private[kafka] object ConfigSettings {
 
   def parseKafkaClientsProperties(config: Config): Map[String, String] = {
-    def collectKeys(c: ConfigObject, prefix: String, keys: Set[String]): Set[String] = {
```

Comment:
Did you see the issue for real, or is this just a nice cleanup?
Wouldn't it have to be a really deep properties structure to be an issue.